Automattic/woocommerce-payments
# Fix issue where checkbox remained disabled when credit card was disabled

Fixes #5525 

#### Changes proposed in this Pull Request

Resolves an issue encountered in the environment where UPE and Credit Card payment method is disabled. Previously, enabling UPE after disabling the credit card resulted in the inability to click on the credit card checkbox.







#### Testing instructions

1. Disable UPE
2. Uncheck CC
3. Enable UPE but don't complete the setup. Close your browser tab when you see this: ![QGbhCe.png](https://user-images.githubusercontent.com/2127470/217126287-0fb3dbc9-4209-4686-b4e3-b2384da8f18b.png)
4. Navigate back to the WCPay settings. Make sure that you can enable Credit Card payment method.

-------------------


**Post merge**
